src/net_stat.cc

```diff
@@ -53,7 +53,6 @@
 #if defined(__linux__)
 #include "linux.h"
 #else
-static char e_iface[50] = "empty";
```

Comment:
Why is this removed? I still see it referenced below.

Reply:
Ah must be because I didn't put the `#if` above in the right place.